namignizer/model.py

```diff
@@ -37,11 +37,11 @@ def __init__(self, is_training, config):
         self._weights = tf.placeholder(tf.float32, [batch_size * num_steps])
 
         # lstm for our RNN cell (GRU supported too)
-        lstm_cell = tf.nn.rnn_cell.BasicLSTMCell(size, forget_bias=0.0)
+        lstm_cell = tf.contrib.rnn.BasicLSTMCell(size, forget_bias=0.0)
         if is_training and config.keep_prob < 1:
-            lstm_cell = tf.nn.rnn_cell.DropoutWrapper(
+            lstm_cell = tf.contrib.rnn.DropoutWrapper(
                 lstm_cell, output_keep_prob=config.keep_prob)
-        cell = tf.nn.rnn_cell.MultiRNNCell([lstm_cell] * config.num_layers)
+        cell = tf.contrib.rnn.MultiRNNCell([lstm_cell] * config.num_layers)
```

Comment:
I think this may be illegal: you should instead create a new lstmcell. @ebrevdo to confirm that I'm not misreading this pattern.